<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SimpleORPEWidget</class>
 <widget class="QWidget" name="SimpleORPEWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>768</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SimpleORPEWidget</string>
  </property>
  <widget class="QListWidget" name="listWidget">
   <property name="geometry">
    <rect>
     <x>15</x>
     <y>20</y>
     <width>191</width>
     <height>691</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pbnExit">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>720</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
  </widget>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>20</y>
     <width>790</width>
     <height>690</height>
    </rect>
   </property>
   <widget class="QWidget" name="page"/>
   <widget class="QWidget" name="page_2"/>
  </widget>
  <widget class="QPushButton" name="pbn_test">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>716</y>
     <width>99</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>PushButton</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
